source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+
+Most of the proposed improvements are in improving existing PHP functionality, especially the new readonly properties/class and the random extension. There were also discussions and proposals about adding a new syntax similar to Swift to declare granular get/set access to properties, bring back the literal-string RFC, passing the calling scope to class magic methods, and more.
+
+Additionally, the [Deprecations for PHP 8.3 RFC](https://wiki.php.net/rfc/deprecations_php_8_3) is also under discussion on features that the community proposes to deprecate in PHP 8.3 and remove in PHP 9.0
+
+
+## Recent RFCs, Merged PRs, Discussions, and Commits
+
+> Changes and improvements to PHP are discussed, reported, and voted on by the PHP Foundation Team, the PHP development team, and contributors. Bug reports are made to the PHP [issue tracker](https://github.com/php/php-src/issues), changes are discussed in [mailing lists](https://www.php.net/mailing-lists.php), minor code changes are proposed as [pull requests](https://github.com/php/php-src/issues), and major changes are discussed in detail and voted on as [PHP RFCs](https://wiki.php.net/rfc). [Documentation](https://github.com/php/doc-en/) and the [php.net website](https://github.com/php/web-php) changes are also discussed and improved at their relevant Git repositories on GitHub.
+
+<br>
+Hundreds of awesome PHP contributors put their efforts into improvements to the PHP code base, documentation, and the php.net website. Here is a summary of some changes made by the people behind PHP. Things marked with 💜 are done by the PHP Foundation team.
+
+### RFC Updates
+
+Following are the RFCs and major pull-requests discussed, voted, and implemented since our last update.
+
+
+
+
+*	**RFC In Voting: [Readonly amendments](https://wiki.php.net/rfc/readonly_amendments) 💜**
+
+	RFC by Nicolas Grekas and Máté Kocsis, attempts to address some of the shortcomings of PHP 8.1 readonly properties and 8.2 readonly classes.
+
+	This RFC proposes allowing `readonly` classes to be extended by non-readonly classes (currently not allowed, and causes a fatal error), and to allow reinitializing readonly properties during cloning (within the `__clone()` magic method).
+
+*	**RFC Accepted: [More Appropriate Date/Time Exceptions](https://wiki.php.net/rfc/datetime-exceptions) 💜**
+
+	RFC by Derick Rethans, proposed to introduce Date/Time extension-specific exceptions and errors. This detailed RFC suggests more specificity in the exceptions with exception classes such as `DateInvalidTimeZoneException`, and `DateMalformedPeriodStringException` as well as promoting some of the current PHP warnings to Error exceptions.
+
+    The vote was accepted, and is pending implementation.
```

Comment:
It's now further then that, there is a PR that's nearly ready to be merged: https://github.com/php/php-src/pull/10366

Reply:
Thank you, I will update it.

Reply:
(and make sure to update the text if the post goes live after the PR is merged).